e2e-tests/e2e-fixtures/dateConstants.js

```diff
@@ -159,6 +159,12 @@ const tomorrowUnix = getUnixTime(tomorrow).toString();
 const todayFormattedTimeHours = format(today, 'h');
 const todayFormattedTimeAmPm = format(today, 'aaa');
 
+/**
+ * Some tests check that validation errors appear,
+ * If a year is entered with zero as a letter. I.e, O, instead of 0.
+ */
+const yearWithZeroLetter = '2O22';
```

Comment:
should this belong here? it all seems to be valid dates formatted in different ways and then this one

Reply:
I highly recommend so. Appreciate all the other dates are properly formatted, but having this value in multiple individual tests makes it unobvious that it is a "strangely formatted" date (0 as O).

@BethThomas141 and I spent a fair amount of time fixing tests that use this format - found it difficult to realise that they have 0 instead of O.

Having it as a DRY variable in `dateConstants` alleviates the issue.

Reply:
Beth is also refactoring the date constants/fixtures in [this PR](https://github.com/UK-Export-Finance/dtfs2/pull/3587)